tests/no_skip_main.nf.test.snap

```diff
@@ -368,13 +368,13 @@
                 "pipeline_info",
                 "pipeline_info/tomte_pipeline_software_mqc_versions.yml",
                 "references",
+                "references/fs1_gffread_output.fa",
```

Comment:
OK, this looks more like an issue in Tomte actually than something that should be fixed by updating the snap-files. Our base dir is named `/fs1` and I suspect it has been previously run with some parameter pointing to a `/home` base dir.

Reply:
Looks like it is coming from here: 

https://github.com/genomic-medicine-sweden/tomte/blob/e355d85120026dc19b97e0ba08f7a1e3549a20c3/workflows/tomte.nf#L65-L66

Reply:
OK, reverting these to `home` which it seems they will end up as when processed in the cloud. This seems like something that can be looked into separately from this PR.